### PR TITLE
Add synthetic event to mark sync status of bridge

### PR DESCRIPTION
**Pull request by weilbith**
Closes: #287

Very open for suggestions of a better name... :see_no_evil: 

The tests fail at the moment due to `gevent` when running the whole suite. I solved it locally with a monkey patch on the very top of the `conftest`. But @schmir I guess your approach with a custom `mypy` script like for the `relay` project is better/cleaner?

@schmir @jannikluhn feel free to reject this PR if it is not necessary anymore to another design change related to the event fetcher.

### CODE REVIEW

File: tools/bridge/bridge/event_fetcher.py [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
 from typing import Any, Dict, List
 
 from eth_utils import to_checksum_address
 from web3 import Web3
 from web3.contract import Contract
 
 
+class FetcherReachedHeadEvent:
+    def __init__(self):
+        self.timestamp = int(time())
```
Comment (jannikluhn) [inline tools/bridge/bridge/event_fetcher.py]: what's the reason to not use `float` here?

Comment (weilbith) [reply tools/bridge/bridge/event_fetcher.py]: I was under the false impression we need the full seconds here. I just remove that. :laughing:

Comment (weilbith) [reply tools/bridge/bridge/event_fetcher.py]: :heavy_check_mark:

**Comment by schmir-at-bb** [comment]
Please do not merge this before #292 has been merged.

**Comment by weilbith** [comment]
Note: I rebase locally on `master` to solve conflicts properly.

**Pull request merged**